crates/pop-parachains/templates/pallet/advanced_mode/src/config_preludes.rs.templ

```diff
@@ -0,0 +1,180 @@
+{{- let using_runtime_event = pallet_common_types|contains(TemplatePalletConfigCommonTypes::RuntimeEvent) -}}
```

Comment:
I've pinged @gupnik to give feedback on this, but he is out for another week or so :)